src/preprocessing/passes/symmetry_breaker.cpp

```diff
@@ -128,27 +137,40 @@ PreprocessingPassResult SymBreakerPass::applyInternal(
 {
   Trace("sym-break-pass") << "Apply symmetry breaker pass..." << std::endl;
   // detect symmetries
-  std::vector<std::vector<Node>> part;
+  std::vector<std::vector<Node>> sterms;
   symbreak::SymmetryDetect symd;
-  symd.getPartition(part, assertionsToPreprocess->ref());
+  symd.computeTerms(sterms, assertionsToPreprocess->ref());
+  if (!sterms.empty())
```

Comment:
Is this supposed to be a placeholder for code to be added later? Otherwise inline this `if` in the trace command